ui/app/scripts/controllers/newDeploymentCtrl.js

```diff
@@ -37,8 +37,15 @@ angular.module('apollo')
 		$scope.possibleGroups = null;
 		$scope.selectedGroups = [];
 		$scope.groupNames = null;
+        $scope.possibleEnvironments = null;
```

Comment:
ident?

Reply:
It seems well indented in my IntelliJ O_O